test/test_autograd.py

```diff
@@ -3216,17 +3216,20 @@ def test_profiler_shapes(self):
 
         print(prof.function_events)
 
-        top_level_expected_events_and_shapes = [
+        second_level_expected_events_and_shapes = [
```

Comment:
what is "second level"?

Reply:
I thought we explicitly want to profile the shape in transpose and addmm. :facepalm: 
Will update.